backend/apis/clients/client-interactions/client-interaction-utils.api.js

```diff
@@ -0,0 +1,32 @@
+const mysql = require("mysql");
+const { responseFullName } = require("../../utils/transform-utils");
+
+const modifiableInteractionTypes = [
+  "clientInteraction:created",
+  "clientInteraction:updated"
+];
+
+exports.createResponseInteractionObject = function createResponseInteractionObject(
+  log
+) {
+  return {
+    id: log.id,
+    title: log.title,
+    serviceId: log.serviceId,
+    interationType: log.interactionType,
+    description: log.description,
+    dateOfInteraction: log.dataOfDescription,
+    duration: log.duration,
+    canModify: modifiableInteractionTypes.some(
```

Comment:
This is wrong. You can modify things that you created and can't modify things that you can't. We should be checking the logged in user for this.

Reply:
I think I am misunderstanding the code on line 47 of activity-log.utils.js

Reply:
Let's delete the entire `canModify` property from clientInteractions. From the API docs and from the api.

Client interactions are always modifiable. We don't need to send a `true` for people to know that. My earlier comment yesterday was me getting confused. But we just need to delete `canModify` entirely since it's not helpful.

Also, btw, this code you have here would completely die if you ran it, since `modifiableInteractionTypes` is not defined.